docs/sphinx/user/inputs_SyntheticTurbulence.rst

```diff
@@ -0,0 +1,77 @@
+Section: SyntheticTurbulence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section is for setting turbulence injection parameters.
+
+.. input_param:: SynthTurb.turbulence_file
+
+   **type:** String, required
+   
+   Name of the netcdf file that contains the data.
+   
+.. input_param:: SynthTurb.wind_direction
+
+   **type:** Real, optional, default = 270.
+   
+   The wind direction. 
+   
+.. input_param:: SynthTurb.grid_location 
+
+   **type:** List of Reals, required
+  
+   Location of the turbulent grid.
+
+.. input_param:: SynthTurb.mean_wind_type
+
+   **type:** String, required 
+  
+   The type of profile to use. Options are ConstValue, LinearProfile, and PowerLawProfile.
+
+.. input_param:: SynthTurb.grid_spacing 
+
+   **type:** Real, optional, default = 2.0
```

Comment:
I don't see where in the code the default is set to 2.0. It seems like only the mean wind type and the time offset have default values. Also, I feel that the gauss smearing factor should be mentioned in the documentation, and it should be noted that even though both the gauss smearing factor and the grid spacing are optional parameters, the gauss smearing factor (if specified) overwrites the effect of the grid spacing input.

Reply:
Good catch, I eliminated this intermediate variable and made the smearing factor a required variable.